### ISSUE TITLE
Link Customization

### ISSUE THREAD

**Issue by yasaspeiris** [beautifulinteractions/beautiful-react-diagrams#106]
**Is your feature request related to a problem? 

The documentation shows how to customize nodes and it works brilliantly. I would also like to modify links, such as changing the fill color, adding an on-hover tool tip to display information etc.

**Describe the solution you'd like**

Include an example on how to modify links in documentation. 

**Describe alternatives you've considered**

I tried modifying the original code but had little success. 

**Additional context**

Thank you for this library! It is absolutely brilliant and easy to use. A few more examples can help beginners like me.

**Comment by allexxis** [comment]
Yes an example on how to modify link will be awesome and state for the animation to be triggered manually could help a lot

**Comment by maofmoran** [comment]
I would also like to be able to customize the links as @yasaspeiris described. It can be very useful.

**Comment by masterofdon** [comment]
Not a real solution maybe but i have added css rules for links. In order to achive that you need to first use className attr of link such as : 

```
links: [
    { input: 'node-1', output: 'node-2', readonly: true, className: "completed-link" },
  ]
```

then created a css rule more specific than original one such as

```
.bi.bi-diagram
  .bi-link-canvas-layer
  .bi-diagram-link.readonly-link.completed-link
  .bi-link-path {
  stroke: blue;
}
```

as I said. not neat but hey it works:D

**Comment by chrisp-jama** [comment]
Related to Link Customization, is it possible to have 2 labels on 1 link path?
